cmd/data-usage-cache.go

```diff
@@ -46,11 +46,14 @@ type sizeHistogram [dataUsageBucketLen]uint64
 //msgp:tuple dataUsageEntry
 type dataUsageEntry struct {
 	// These fields do no include any children.
-	Size     int64
-	Objects  uint64
-	ObjSizes sizeHistogram
-
-	Children dataUsageHashMap
+	Size                   int64
```

Comment:
`dataUsageCacheVer` must be bumped since this is incompatible with previous versions.

`//msgp:tuple dataUsageEntry` makes it encode it without keys, as a fixed size array.